TrackletGraph.py

```diff
@@ -167,9 +170,32 @@ def populate_bitwidths(mem,hls_dir): # FIXME this information should be parsed f
         for item in diskList:
             disk = max(disk,mem.inst.find(item))
         if mem.mtype == "VMStubsTE":
-            mem.bitwidth = 22 if mem.inst.find("L1") else 16 # FIXME
-        elif mem.mtype == "AllStubs":
+            phi_region = mem.inst.split("PHI")[1][0]
+            position = mem.inst.split("_")[1][:2]        
+            # Determine if TE outer/inner/overlap. MAKE THIS PRETTIER
+            # VMStubTEInner: 22/23 bits
```

Comment:
Can VMStubTEInner logic be simpler? i.e.
if position == 'L5':
  mem.bitwidth = 23
else
  mem.bitwidth = 22

Reply:
Not really, because we need to figure out if the "VMStubsTE" belongs to Inner/Outer/Overlap and that depends on the layer number and the phi region.